doc/source/development/dev_environment.rst

```diff
@@ -136,3 +136,36 @@ From a Conda enabled console
         cd c:\dev\GDAL
         cd _build.vs2019
         ctest -V --build-config Release
+
+
+.. _setting_dev_environment_variables:
+
+Setting development environment variables
+-----------------------------------------
+
+Once GDAL has been built, a number of environment variables must be set to be
+able to execute C++ or Python utilities of the build directory, or run tests.
```

Comment:
This isn't necessary to run tests via `ctest`, but I don't know if it's worth being that specific.